### PR TITLE
refactor: Set user's market from environment variable

### PR DESCRIPTION
**Pull request by seckin206**

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -77,11 +77,11 @@ export const getAddCredentialsLink = (
     `client_id=${process.env.REACT_APP_TINK_LINK_PERMANENT_USERS_CLIENT_ID}`,
     "redirect_uri=http://localhost:3000/callback",
     "scope=user:read,credentials:read",
-    "market=SE",
+    `market=${process.env.REACT_APP_TINK_LINK_PERMANENT_USERS_MARKET}`,
     "locale=en_US",
     `state=${userId}`,
     `authorization_code=${authorizationCode}`,
-    "test=true" // Change this to `false` if you want to see real banks instead of test providers.
+    "test=false" // Change this to `false` if you want to see real banks instead of test providers.
```
Comment (seckin206) [inline src/api/index.ts:84]: oops, will remove that

**Comment by seckin206** [comment]
@fredrikssongustav please have another look 🙂

**Comment by fredrikssongustav** [comment]
> please have another look 🙂 

Sorry for being picky about this, but I think we should avoid setting the `*_CLIENT_ID` and `*_CLIENT_SECRET` in the dotenv configuration. I think it's a too convenient tripwire to expose your credentials and force rotation.

**Pull request merged**